api/MouseEvent.json

```diff
@@ -452,48 +881,99 @@
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "long_to_double": {
```

Comment:
The MDN table only mentions `MouseEvent.screenX`, `MouseEvent.screenY`, `MouseEvent.clientX`, and `MouseEvent.Y` [sic] as being converted from a long to a double. Nothing about offset as far as I can tell.

Reply:
There was a [note](https://developer.mozilla.org/docs/Web/API/MouseEvent/offsetY#Browser_compatibility) about it.